EIPS/eip-6785.md

```diff
@@ -0,0 +1,203 @@
+---
+eip: 6785
+title: ERC-721 Utilities Extension
+description: Easy access to utility information via functions and the NFT's metadata
+author: Otniel Nicola (@OT-kthd), Bogdan Popa (@BogdanKTHD)
+discussions-to: https://ethereum-magicians.org/t/eip-6785-erc-721-utilities-extension/13568
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-27
+requires: 165, 721
+---
+
+## Abstract
+
+This specification defines standard functions that outline what a utility entails for a specific NFT and how it may be
+used. This specification is an optional extension of [ERC-721](./eip-721.md).
```

Comment:
You should make it extremely obvious in the abstract that a "utility" is a property of a token, and that you aren't defining "utility functions for interacting with tokens."